scipy/stats/_stats_py.py

```diff
@@ -4674,21 +4671,23 @@ def fisher_exact(table, alternative='two-sided'):
         Acute montain sickness             7           17
         No                                15            5
 
+
     Is there evidence that the acetazolamide 250 mg reduces the risk of
     acute montain sickness?
     We begin by formulating a null hypothesis :math:`H_0`:
 
-        The effect of acetazolamide is equivalent to that of placebo.
+        The effect of acetazolamide is equivalent to that of placebo and
+        does not reduce the risk of acute montain sickness.
```

Comment:
"reduce the risk" suggests that we will examine the "relative risk". The statistic reported by `fisher_exact` is the "odds ratio", so I think the statement should be about the odds rather than the risk. The difference between odds ratio and relative risk is already a common source of confusion, so I think we should be careful not to exacerbate it.

Reply:
Ok so `reduce the odds`?

Reply:
I guess that's fine. I had in mind to interpret the statistic returned by `fisher_exact` (the odds ratio) and not just the p-value. But let's not increase the scope.

Reply:
So how about 

> The odds of experiencing acute mountain sickness are the same with the acetazolamide treatment as they are with placebo.

against the alternative:

> The odds of experiencing acute mountain sickness with acetazolamide treatment are *less* than the odds of experiencing acute mountain sickness with placebo.

I highlight *less* here to imply the connection between this statement and `alternative='less'`.